packages/cxl-ui/src/components/cxl-adeft-star-rating.js

```diff
@@ -0,0 +1,52 @@
+import { customElement, html, LitElement, property } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+import '@cwmr/iron-star-rating';
+
+@customElement('cxl-adeft-star-rating')
+export class CXLAdeftStarRating extends LitElement {
```

Comment:
Why are we not extending `IronStarRating` class directly? We probably should.

https://github.com/chadweimer/iron-star-rating/blob/a76f8bd38ecd8eab535bbf3488af0ca6100a1523/iron-star-rating.js#L12

Reply:
> Why are we not extending `IronStarRating` class directly? We probably should.
> 
> https://github.com/chadweimer/iron-star-rating/blob/a76f8bd38ecd8eab535bbf3488af0ca6100a1523/iron-star-rating.js#L12

I think this is impossible since there is no `export` class in this file. Is there a possibility to extend the class that is not exported?

Reply:
That's a good question, I don't actually know, filed https://github.com/chadweimer/iron-star-rating/issues/11

Itmw, we could test by monkey-patching iron-star-rating.js ourselves with the `export` statement, and see if we can make our functionality work.

Reply:
`cxl-star-rating` converted to polymer, and `iron-star-rating.js` "monkey patched" with `export` added, seems it works https://github.com/conversionxl/aybolit/pull/88/commits/de171d4ac4f1e7f1a9236760bbe95b5f8b61fe98